src/winter_onboarding_2021/fleet_management_service/views/layout.clj

```diff
@@ -2,12 +2,14 @@
   (:use [hiccup.page :only (html5 include-css include-js)]
         [winter-onboarding-2021.fleet-management-service.views.content :as content]))
 
-(defn application [title & content]
+(defn application [request title & content]
   (html5 [:head
           [:title title]
           [:link {:rel "icon" :href "public/favicon.ico" :type "image/x-icon"}]
-          (include-css "public/css/bootstrap.min.css")
-          (include-js "public/js/jquery-3.6.0.min.js")
-          (include-js "public/js/bootstrap.bundle.min.js")
+          (include-css "/public/css/bootstrap.min.css")
```

Comment:
What's the difference between "/public/..." and "public/..."

Reply:
Adding a `/` before `public` tells the browser that the path is not relative.

If we are not adding the `/` in the above lines and if I were in, say, `/cabs` URL in the browser then browser will try to find the bootstrap files in `/cabs/public/*`